odoo/addons/base/tests/test_views.py

```diff
@@ -4022,6 +4023,359 @@ def test_render_all_views(self):
+            'quality',
+            'quality_control_iot',
+            'rating',
+            'sale_stock_renting',
+            'repair',
+            'partner_autocomplete',
+            'pos_restaurant_appointment',
+            'sale_pdf_quote_builder',
+            'base_geolocalize',
+            'website_sale_stock',
+            'account_qr_code_emv',
+            'account_avatax_geolocalize',
+            'web_map',
+            'auth_signup',
+            'mail_bot_hr',
+            'hr_maintenance',
+            'resource',
+            'room',
+            'sale_loyalty',
+            'sale_purchase',
+            'sale_stock',
+            'sale_external_tax',
+            'account_avatax_sale',
+            'sale_management',
+            'partner_commission',
+            'spreadsheet_dashboard_sale_subscription',
+            'account_sepa_direct_debit',
+            'sign',
+            'website_slides',
+            'website_slides_survey',
+            'mass_mailing_slides',
+            'website_sale_slides',
+            'snailmail',
+            'social',
+            'social_youtube',
+            'social_crm',
+            'spreadsheet_dashboard_edition',
+            'stock_picking_batch',
+            'quality_control_picking_batch',
+            'stock_barcode_picking_batch',
+            'delivery_shiprocket',
+            'stock_barcode_product_expiry',
+            'stock_intrastat',
+            'uom',
+            'auth_totp',
+            'utm',
+            'base_vat',
+            'website_twitter_wall',
+            'social_push_notifications',
+            'website_sms',
+            'website_studio',
+            'whatsapp',
+            'hr_skills_slides',
+            'website_payment',
+            'voip',
+            'social_instagram',
+            'payment_custom',
+            'hr_hourly_cost',
+            'helpdesk_repair',
+            'product_margin',
+            'hr_contract_salary_holidays',
+            'account_audit_trail',
+            'product_email_template',
+            'stock_enterprise',
+            'documents_l10n_be_hr_payroll',
+            'l10n_be_reports',
+            'l10n_be_soda',
+        )
+
+        module_log_views = defaultdict(list)
+        module_error_views = defaultdict(list)
+        uncommented_regexp = r'''(<field [^>]*invisible=['"](True|1)['"][^>]*>)[\s\t\n ]*(.*)'''
+        views = self.env['ir.ui.view'].search([('type', 'in', ('tree', 'form')), '|', ('arch_db', 'like', 'invisible=_True_'), ('arch_db', 'like', 'invisible=_1_')])
+        for view in views:
+            for field, _val, comment in re.findall(uncommented_regexp, view.arch_db):
+                if (not comment or not comment.startswith('<!--')) and view.model_data_id:
+                    views = module = view.model_data_id.module
+                    if module in only_log_modules:
+                        module_log_views[module].append(view.model_data_id.name)
```

Comment:
I would add a check to see that at least one error was found in this module, and maybe add an error if it wasn't the case to force devs to remove modules from the list once fixed. Do you think it is possible or it will depends to much on installed modules?